retrieval/resource_map.go

```diff
@@ -77,6 +77,17 @@ var GCEResourceMap = ResourceMap{
 	},
 }
 
+var AnthosL4LBMap = ResourceMap{
+	Type: "anthos_l4lb",
+	LabelMap: map[string]labelTranslation{
+		ProjectIDLabel:                         constValue("project_id"),
+		KubernetesLocationLabel:                constValue("location"),
+		"__meta_kubernetes_service_annotation_anthos_l4lb_kind":          constValue("kind"),
+		"__meta_kubernetes_service_annotation_anthos_l4lb_group_name":          constValue("group_name"),
+		"__meta_kubernetes_service_annotation_anthos_l4lb_hostname":           constValue("hostname"),
+	},
+}
+
```

Comment:
Any reason you added this here, and not at the end (just before the declaration of `ResourceMapList`)?